Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -497,6 +498,42 @@ mode would look as follows:
         --set k8sServiceHost=REPLACE_WITH_API_SERVER_IP \\
         --set k8sServicePort=REPLACE_WITH_API_SERVER_PORT
 
+
+.. _DSR mode with IPIP:
+
+Direct Server Return (DSR) with IPIP
```

Comment:
Shouldn't this section be right after the ` Direct Server Return (DSR) with IP option` section?

Reply:
Sure. Will do.